package.json

```diff
@@ -31,7 +31,7 @@
     "docs:diagram": "nomnoml docs/diagrams/architecture.noml > docs/assets/architecture.svg",
     "docs:summary": "./scripts/architecture-summary.sh > ARCHITECTURE.md",
     "clean": "rimraf apps/*/packages/*",
-    "postinstall": "(cd apps/server && npm i); (cd apps/ui && npm i)",
```

Comment:
I find this quite useful during local development so I can get all `node_modules` set up with a single command.

Reply:
I'm sure, this could be replaced by `npm run install:all` or something similar.
We need a different strategy for the SUT container - right now it has to build the entire stack and is easily the slowest container to build (~5m). This PR starts to reduce some of the overhead.

Reply:
I've added an `install` script that does the same thing